src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/course/domain/PrerequisiteUpdate.kt

```diff
@@ -1,6 +1,6 @@
 package uk.gov.justice.digital.hmpps.hmppsaccreditedprogrammesapi.course.domain
 
-data class NewPrerequisite(
+data class PrerequisiteUpdate(
```

Comment:
I don't think we have the ability to update Prerequisites, I think we just create or remove them. I wonder if `CreatedPrerequisite`  or `NewPrerequisite` captures that a bit more clearly? I may be wrong about this though.